.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+---
+name: "\U0001F41B Bug fix"
+about: Create a report to help us improve
+labels: bug
```

Comment:
Should it be something else (PR related) other than bug?

Reply:
That is removed now, was left in by accident.